core/src/main/resources/jenkins/management/AdministrativeMonitorsConfiguration/config.groovy

```diff
@@ -41,6 +41,9 @@ f.section(title: _("Administrative monitors configuration")) {
                                 title: am.displayName,
                                 checked: am.enabled,
                                 json: am.id)
+                        if (am.isSecurity()) {
+                            span(style: 'border: 1px solid grey; color: grey; border-radius: 4px; padding: 3px; margin-left: 5px;', _("Security"))
```

Comment:
could we add a css class for this?

Reply:
🤷 sure

Reply:
Makes sense, let me know if you would like a hand like we did with the plugin manager badges.

Reply:
@fqueiruga Sure, any help to not make this look ugly is appreciated.

Reply:
Sure, just let me know once the PR is ready 👍

Reply:
Fixed.